src/components/RichText.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Editor } from '@tinymce/tinymce-react';
+
+
+class RichText extends React.Component {
+  handleEditorChange = (e) => {
+    console.log(
+      'Content was updated:',
+      e.target.getContent()
+    );
+  }
+
+  render() {
+    return (
+        <div className="Editor">
+      <Editor
+      apiKey="o8rmdhfx3vylfdbivbn0pbc26qo09czj8vabx01vgy5qcfte"
```

Comment:
API key should not be used publicly, especially, it should not be pushed to GitHub. Save it as a variable in a private file and use that variable here.